debug_toolbar/stubs.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any, List, NamedTuple, Optional, Tuple
+
+from django import template as dj_template
```

Comment:
@matthiask Admittedly I haven't done typing in a project of my own that requires a stubs file. I'm unsure how I feel about a single stub file without docs or some way to determine if they're used someplace. However, I think I'm okay with us using this until we find it doesn't work.

Thoughts?